src/views/SpaceSettings.vue

```diff
@@ -361,5 +371,22 @@ const isViewOnly = computed(() => {
         </BaseInput>
       </div>
     </ModalConfirmAction>
+    <ModalNotice
```

Comment:
Something weird happening here on small screen with the line break. Can you reproduce? 
<img width="443" alt="image" src="https://user-images.githubusercontent.com/51686767/230577267-f3603324-cb42-4482-b823-86dc1c77bc64.png">

Reply:
Will remove br and v-html line breaks good without it